src/patternfly/components/DataToolbar/examples/data-toolbar-stacked-example.hbs

```diff
@@ -75,20 +75,35 @@
   {{/data-toolbar-content}}
 
   {{#> data-toolbar-expandable-content data-toolbar-expandable-content--modifier="pf-m-reveal-on-xl"}}
-    {{#> data-toolbar-item}}
-      {{#> select id=(concat data-toolbar--id '-select-checkbox-resource') select--IsCheckboxSelect="true"}}
-        Pod
-      {{/select}}
-    {{/data-toolbar-item}}
-    {{#> data-toolbar-item}}
-      {{#> select id=(concat data-toolbar--id '-select-checkbox-status') select--IsCheckboxSelect="true"}}
-        Running
-      {{/select}}
-    {{/data-toolbar-item}}
-    {{#> data-toolbar-item}}
-      {{#> select id=(concat data-toolbar--id '-select-checkbox-type') select--IsCheckboxSelect="true"}}
-        Any
-      {{/select}}
-    {{/data-toolbar-item}}
+      {{#> data-toolbar-group newcontext}}
```

Comment:
Line [8](https://github.com/patternfly/patternfly-next/pull/2229/files#diff-8af5b5bc5ea0f51d0c1286e2bd8baf7bR8), [18](https://github.com/patternfly/patternfly-next/pull/2229/files#diff-8af5b5bc5ea0f51d0c1286e2bd8baf7bR18), [28](https://github.com/patternfly/patternfly-next/pull/2229/files#diff-8af5b5bc5ea0f51d0c1286e2bd8baf7bR28), here at [78](https://github.com/patternfly/patternfly-next/pull/2229/files#diff-8af5b5bc5ea0f51d0c1286e2bd8baf7bR78), [88](https://github.com/patternfly/patternfly-next/pull/2229/files#diff-8af5b5bc5ea0f51d0c1286e2bd8baf7bR88), and [98](https://github.com/patternfly/patternfly-next/pull/2229/files#diff-8af5b5bc5ea0f51d0c1286e2bd8baf7bR98) would all be
```
{{#> data-toolbar-group newcontext data-toolbar-group--modifier="pf-m-shared-item"}}
```